src/code/examples/assets-ready.rs

```diff
@@ -1,45 +1,35 @@
-use bevy::prelude::*;
+use bevy::{asset::HandleId, prelude::*};
 
+#[derive(Default)]
 // ANCHOR: example
-struct AssetsLoading(Vec<HandleUntyped>);
+struct AssetsLoading(Vec<HandleId>);
```

Comment:
This change actually introduces a bug. Bevy will unload the asset since there is no strong handle anymore. See bevyengine/bevy#2347

Reply:
Yikes. I'm... very sorry for that :flushed: I'll update the code :grimacing:.

Reply:
Update the code according to the issue mentioned; verified the logic on the linked repository. thanks! :grin: